FogCreek/Glitch-Community
# Solar address: only admins should be able to delete a project

## Links
* https://solar-address.glitch.me
* https://glitch.manuscript.com/f/cases/3328354/project-MEMBERs-can-undelete-projects-but-can-t-delete-them

## Changes:
* should only be able to delete projects that you are the admin on. Pretty sweet now I won't worry about accidentally deleting community lol. 

## How To Test:
* go to https://solar-address.glitch.me
* log in with your token
* visit the user page verify and check out some projects that you have joined but you are not the original creator for, notice that you only see delete projects when you are the original creator
* this popover shows up in a lot of places, feel empowered to check out collections, teams, project pages, etc!

## Feedback I'm looking for:
* did I miss anything?

## Things left to do before deploying:
- [x] get someone to verify my logic and that projects seem appropriate for them.
